integration_tests/CMakeLists.txt

```diff
@@ -101,6 +101,9 @@ macro(RUN_UTIL RUN_FAIL RUN_NAME RUN_FILE_NAME RUN_LABELS RUN_EXTRAFILES RUN_NOM
             set_target_properties(${name} PROPERTIES LINKER_LANGUAGE C)
             target_link_libraries(${name} lpython_rtlib)
             add_test(${name} ${CMAKE_CURRENT_BINARY_DIR}/${name})
+            add_test(
+                NAME "${name}_jit"
+                COMMAND ${LPYTHON} --jit ${extra_args} ${CMAKE_CURRENT_SOURCE_DIR}/${file_name}.py)
```

Comment:
Are you facing any issue/challenges in implementing the approach shared in https://github.com/lcompilers/lpython/pull/2564#pullrequestreview-1926239513?

Reply:
If I use that approach, then `RUN(NAME jit_01 LABELS cpython llvm EXTRA_ARGS --jit)` with

```cmake
add_custom_command(
                OUTPUT ${name}.o
                COMMAND ${LPYTHON} -c ${extra_args} ${CMAKE_CURRENT_SOURCE_DIR}/${file_name}.py -o ${name}.o
                DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/${file_name}.py
                VERBATIM)
```

will produce

```cmake
add_custom_command(
    OUTPUT test_file.o
    COMMAND lpython -c --jit path/to/test_file.py -o test_file.o
    ...)
```

But while using `--jit` no `.o` file is generated. And `cmake` complains about it. Also, that would not technically be testing the JIT. It will fail immediately and exit the remaining tests because it would be part of the build step, not the test step.

I hope you could understand my explanation. Let me know if there are things in my explanation you could not understand.

Reply:
Oh, I see. Thanks for the explanation. I think then a better/simpler approach to test this could/would be by adding it as a reference test.

We might also be able to test this by using tests in `src/lpython/tests`, but that might be slightly involved. I think adding it as a reference test is a better approach.